### PR TITLE
Update Wear OS docs for showing units in complication

### PR DESCRIPTION
**Pull request by slovdahl**
Related to home-assistant/android#3703.

### CODE REVIEW

**Comment by slovdahl** [comment]
Not sure if this really is necessary @jpelgrom?

**Comment by jpelgrom** [comment]
> Not sure if this really is necessary @jpelgrom?

Yes typically features like these are documented. Show entity name is a bit hidden ('depends on the watchface' but also your configuration). For consistency, maybe you could add that as well (optionally the name and unit of measurement)?

**Comment by dshokouhi** [comment]
Don't forget to add a beta label for this feature :)

File: docs/wear-os/wear-os.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ You may use HTML to format the text displayed. The following tags are currently
 
 ## Complications
 
-* An entity state complication can be displayed on your watchface. The complication will display the current state of the selected entity. Depending on the watchface, the complication may also show the entity name and icon, and supports 'short text' and 'long text' complication types.
+* An entity state complication can be displayed on your watchface. The complication will display the current state of the selected entity, and optionally the unit of the measurement. Depending on the watchface, the complication may also show the entity name and icon, and supports 'short text' and 'long text' complication types.
```
Comment (slovdahl) [inline docs/wear-os/wear-os.md:66]: Like this?

```suggestion
* An entity state complication can be displayed on your watchface. The complication will display the current state of the selected entity, and optionally the name and unit of the measurement. Depending on the watchface, the complication may also show the entity name and icon, and supports 'short text' and 'long text' complication types.
```

**Comment by slovdahl** [comment]
> Don't forget to add a beta label for this feature :)

Good point. Is that something I can do?

**Comment by dshokouhi** [comment]
> > Don't forget to add a beta label for this feature :)
> 
> Good point. Is that something I can do?

yes you can. You just need to make sure the added text has the beta label applied before it so its clear that feature is for the beta. Might be easier to add a sub bullet  and apply the label before the text changes. Later on when we cut a production release we will remove the labels and merge things.

`<span class='beta'>BETA</span>` is the beta label to apply

**Comment by slovdahl** [comment]
Ah, thanks!

It's not pretty, but is it it good enough?

image

**Comment by jpelgrom** [comment]
Shouldn't "of the measurement" be included in the brackets, as it is a [unit of measurement]?

**Comment by slovdahl** [comment]
Yes :see_no_evil: Thanks for reading thoroughly.

**Review by jpelgrom — APPROVED** [review]
Looks good, can be merged when Android PR is merged

**Review by JBassett — APPROVED** [review]

**Pull request merged**